Detectors/MUON/MCH/Raw/Decoder/include/MCHRawDecoder/DataDecoder.h

```diff
@@ -147,19 +154,21 @@ class DataDecoder
 
   o2::mch::raw::PageDecoder mDecoder; ///< CRU page decoder
 
-  RawDigitVector mDigits; ///< vector of decoded digits
-
+  RawDigitVector mDigits;                               ///< vector of decoded digits
   std::unordered_set<OrbitInfo, OrbitInfoHash> mOrbits; ///< list of orbits in the processed buffer
-  SampaTimeFrameStarts mSampaTimeFrameStarts;           ///< time stamps of the TimeFrames in the processed buffer
+
+  std::optional<uint32_t> mFirstOrbitInRun{0}; ///< first orbit in the processed run
+  std::optional<uint32_t> mFirstOrbitInTF{0};  ///< first orbit in the processed time frame
```

Comment:
this data member mFirstOrbitInTF is not really needed since it is just copied into mSampaTimeFrameStart.mOrbit, which is the variable used later on. Beside, the std::optional is not really used since the validity of mSampaTimeFrameStart.mOrbit (or mFirstOrbitInTF) is never checked.

Reply:
I agree, removed.